packages/@aws-cdk/aws-servicecatalog/lib/private/association-manager.ts

```diff
@@ -179,6 +167,32 @@ export class AssociationManager {
     };
   }
 
+  private static setLaunchRoleConstraint(portfolio: IPortfolio, product: IProduct, options: CommonConstraintOptions,
+    roleOptions: LaunchRoleConstraintRoleOptions): void {
```

Comment:
This formatting blends the signature and the body of the function too much. How about

```suggestion
  private static setLaunchRoleConstraint(
    portfolio: IPortfolio, product: IProduct, options: CommonConstraintOptions,
    roleOptions: LaunchRoleConstraintRoleOptions,
  ): void {
```

instead?